lib/class-wp-theme-json-gutenberg.php

```diff
@@ -344,9 +344,10 @@ class WP_Theme_JSON_Gutenberg {
 	 * @since 6.1.0 Added `layout.definitions` and `useRootPaddingAwareAlignments`.
 	 * @since 6.2.0 Added `dimensions.minHeight`, 'shadow.presets', 'shadow.defaultPresets',
 	 *              `position.fixed` and `position.sticky`.
-	 * @since 6.3.0 Removed `layout.definitions`. Added `typography.writingMode`.
+	 * @since 6.3.0 Removed `layout.definitions`.
 	 * @since 6.4.0 Added `layout.allowEditing`.
-	 * @since 6.4.0 Added `lightbox`.
+	 *              Added `lightbox`.
+	 *              Added `typography.WritingMode`.
```

Comment:
I removed the text-orientation here earlier because I did not know it was still considered for 6.4 after beta 1.